setup.py

```diff
@@ -5,7 +5,7 @@
 except ImportError:
     from distutils.core import setup
 
-version = '0.5.7'
+version = '0.5.9'
```

Comment:
I'm guessing this means we never released 0.5.8, we'll want to do that now if that is the case.